recipe/meta.yaml

```diff
@@ -1,67 +1,67 @@
 {% set name = "fastparquet" %}
-{% set version = "0.5.0" %}
-{% set sha256 = "3b558059327dce2c73a775aa2f2029e052fcf6e61880e15681e04ede5f0f20b7" %}
+{% set version = "2023.4.0" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/f/{{ name }}/{{ name }}-{{ version  }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: 917e6d288ea07e10b28b5fa4b4c0b70f60b14971ece3ba5bf30690320a53aa70
 
 build:
-  number: 2
-  # Currently it's not available on s390x:
-  # There is a missing numba package.
-  skip: True  # [py<36 or s390x]
+  number: 0
+  #skip s390x because of lack of cramjam and numba
+  skip: true  # [py<38 or s390x]
   script:
     - del fastparquet\*.c  # [win]
     - rm fastparquet/*.c   # [unix]
-    - {{ PYTHON }} -m pip install . --no-deps -vv
+    - "{{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv "
 
 requirements:
   build:
     - {{ compiler('c') }}
   host:
     - python
-    - cython
-    - numba
-    - numpy
-    - packaging
-    - pandas
     - pip
-    - pytest-runner
+    - packaging
+    - wheel
     - setuptools
-    - thrift >=0.11
+    - numpy 
+    - setuptools-scm 7.0.4
+    - cython 0.29.32
+    - pandas 1.5.2
```

Comment:
`pandas` and `packaging` aren't required for `host`, see https://github.com/dask/fastparquet/blob/c8028e00de2e5cb8594c442d2c1837019a61553a/pyproject.toml#L2 or https://github.com/dask/fastparquet/blob/c8028e00de2e5cb8594c442d2c1837019a61553a/setup.py#L56-L61

Reply:
done